src/cli/project/channel/list.rs

```diff
@@ -0,0 +1,28 @@
+use crate::Project;
+use clap::Parser;
+
+/// List the channels in the project file.
+#[derive(Parser, Debug, Default)]
+pub struct Args {
+    /// Whether to display the channel's names or urls
+    #[clap(long)]
+    pub urls: bool,
+}
+
+pub async fn execute(project: Project, args: Args) -> miette::Result<()> {
+    project.channels().iter().for_each(|channel| {
+        if args.urls {
+            // Print the channel's url
+            eprintln!("{}", channel.base_url());
```

Comment:
This list feature, and all list features for that matter, might be used in code so I would make them `println` instead of `eprintln` so they will just to to the `stdout`.

I'm not going to comment "Same" on all the eprintln's. But a good rule of thumb would be if the info would be machine readable and it is the result of what you wanted the cli to do then you can use `println`. If it's info use `eprintln`. We use the `tracing` library for warnings, debug and errors.

Reply:
Yes, it makes sense.